tests/sentry/api/endpoints/relocations/test_index.py

```diff
@@ -69,16 +69,65 @@ def test_success_simple(self, uploading_complete_mock):
                                 ).getvalue(),
                                 content_type="application/tar",
                             ),
-                            "orgs": ["testing", "foo"],
+                            "orgs": "testing, foo",
                         },
                         format="multipart",
                     )
 
         assert response.status_code == 201
         assert Relocation.objects.count() == relocation_count + 1
         assert RelocationFile.objects.count() == relocation_file_count + 1
+        assert Relocation.objects.get(owner_id=self.owner.id).want_org_slugs == ["testing", "foo"]
         assert uploading_complete_mock.call_count == 1
 
+    # pytest parametrize does not work in TestCase subclasses, so hack around this
+    for org_slugs, expected in [
```

Comment:
Could probably split this into 2 table-driven tests, one for success cases, and one for failure cases due to one or more invalid slug.